### PR TITLE
App Release 1

### PR DESCRIPTION
**Pull request by raminka13**
## Leagues Standings App v.0.1

 - Selected the [API](https://github.com/azharimm/football-standings-api).
 - Built file structure for React app.
 - Prepared routes and navigation in-app.
 - Fetch data from the API in the reducers thunk.
 - Created the tests for the application.
 - Style your components to match the design by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
 - Deployed the project to [Netlify](https://leagues-standings-raminka.netlify.app/) and tested for final details.
 - Recorded a [Video Presentation](https://youtu.be/Hy242UqNOts) for project.
 - Created a great `README` and `PR` description.

I hope you enjoy this project as much as I enjoyed coding it.
Thanks.

### CODE REVIEW

File: src/components/Standings.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <tr id="head">
+            <th>Pos</th>
+            <th> </th>
+            <th>Name</th>
+            <th>GP</th>
+            <th>W</th>
+            <th>D</th>
+            <th>L</th>
+            <th>Points</th>
+          </tr>
```
Comment (thecodechaser) [inline src/components/Standings.js:33]: - [ ] Please implement a "Back" button (<) on the details page, When the user clicks on the button the user should navigate to the home page. `(I can see there's a home button on the navbar that can be used to go to the home page but this is a project requirement to implement the back button)`.
image

File: src/components/Seasons.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          Premier League Standings
+        </h2>
+      </div>
+      <ul className="seasons-list">
+        {seasonsList.map((season) => (
+          <li key={season.year} className="season-button">
+            <Link to="standings" value={season.year} state={season.year} className="season-year">
+              {season.year}
+            </Link>
+            <span className="season-display">
```
Comment (thecodechaser) [inline src/components/Seasons.js]: - [ ] Please implement the search functionality on the homepage that should filter the data, You can filter the year on the search functionality. `(For example, users can search the year 2015 in the search bar)`.
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @raminka13👋, 

### You are doing a marvelous job on the React capstone project and this is one of the best projects I have seen so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Awesome UI design. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Standings.js (hunk 1 above)
Comment (raminka13) [reply src/components/Standings.js:33]: Instead of home button just a back button?

File: src/components/Seasons.js (hunk 2 above)
Comment (raminka13) [reply src/components/Seasons.js]: Awesome suggestion. Will do that.

File: src/components/Standings.js (hunk 1 above)
Comment (thecodechaser) [reply src/components/Standings.js:33]: @raminka13 Home button should be there as well

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @raminka13  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**